### DISCUSSION TITLE
ERROR: No matching distribution found for sqlite3

### DISCUSSION THREAD

**Discussion by akaday** [community/community#140700] [Packages]
### Select Topic Area

Show & Tell

### Body

 pip install -r requirements.txt
Defaulting to user installation because normal site-packages is not writeable
Collecting sqlalchemy (from -r requirements.txt (line 1))
  Downloading SQLAlchemy-2.0.35-cp312-cp312-win_amd64.whl.metadata (9.9 kB)
Collecting psycopg2-binary (from -r requirements.txt (line 2))
  Downloading psycopg2_binary-2.9.9-cp312-cp312-win_amd64.whl.metadata (4.6 kB)
Collecting mysql-connector-python (from -r requirements.txt (line 3))
  Downloading mysql_connector_python-9.0.0-cp312-cp312-win_amd64.whl.metadata (2.0 kB)
ERROR: Could not find a version that satisfies the requirement sqlite3 (from versions: none)
ERROR: No matching distribution found for sqlite3

**Comment by varunpareek690** [comment]
This is because sqlite3 is already installed in python , and reinstalling it using pip won't work
Try this in shell and see if it's installed

```bash
python
```
(Python Terminal opens up)

```python
import sqlite3
print(sqlite3.sqlite_version)
```
